ckanext/scheming/templates/scheming/form_snippets/slug.html

```diff
@@ -32,4 +32,4 @@
     is_required=h.scheming_field_required(field)
     ) }}
 
-
+{%- snippet 'scheming/form_snippets/help_text.html', field=field -%}
```

Comment:
was this getting repeated?

Reply:
It was added not removed, no help text was being displayed for slugs.